lib/search.rb

```diff
@@ -847,6 +851,23 @@ def posts_query(limit, opts = nil)
     posts.limit(limit)
   end
 
+  def categories_ignored(posts)
+    posts.where(<<~SQL, Searchable::PRIORITIES[:ignore])
```

Comment:
I don't know, this feels way too fancy to me and uneeded. 

Force people to explicitly set on each category and child category. I can see cases where people may for whatever reason ignore parent, but keep children, nothing wrong with that.

Reply:
@SamSaffron If I do it in Ruby the code is actually quite simple but it comes at the expense of two extra connections to the DB. 

> Force people to explicitly set on each category and child category. I can see cases where people may for whatever reason ignore parent, but keep children, nothing wrong with that.

Not sure about this. If a parent category has 10 child category, it'll be quite troublesome to have to set ignore on the 10 child category. Ignoring parent but keeping children seems like a case where the parent category has very little relationship to the children categories.

Reply:
Oh the SQL here is 100% fine for the proposed feature. BUT I want to keep this simple for now force people to be explicit per category with no inheritance.

Cause otherwise is it going to get crazy when you have other things except for "ignore"

low priority parent
   nothing set on child 1
   high priority on child 2 

Good luck explaining what happens here :)

Reply:
Ah OK :+1: This is done.